src/DotNet.Status.Web/Startup.cs

```diff
@@ -80,6 +80,14 @@ public void ConfigureServices(IServiceCollection services)
                     .ProtectKeysWithAzureKeyVault(kvClient, key.KeyIdentifier.ToString())
                     .SetDefaultKeyLifetime(TimeSpan.FromDays(14))
                     .SetApplicationName(typeof(Startup).FullName);
+
+                services.AddHsts(options =>
```

Comment:
I think we need to not do this for local, since local + HSTS = really bad day for a long time that you can't really fix, because all "localhost" requests get locked into a broken state, since it's by domain.  @alexperovich or @jonfortescue did it for all our other services, so might know more? Maybe I'm misremembering something.

Reply:
yes, this is already inside the "else" clause of `if (Env.IsDevelopment())`, is that not what you're asking for ?

Reply:
Oh, missed the block. NEvermind.